### ISSUE TITLE
error while 

### ISSUE THREAD

**Issue by gamersindo1223** [realcoloride/node_characterai#135]
[node_characterai] Puppeteer - This is an experimental feature. Please report any issues on github.
/root/rest-api/node_modules/@puppeteer/browsers/lib/cjs/launch.js:262
                reject(new Error([
                       ^

Error: Failed to launch the browser process!
/root/.cache/puppeteer/chrome/linux-1108766/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory

TROUBLESHOOTING: https://pptr.dev/troubleshooting

    at Interface.onClose (/root/rest-api/node_modules/@puppeteer/browsers/lib/cjs/launch.js:262:24)
    
    
    Basically the error above appeared only on my vps, but when I ran the same code on my another vps (azure) i didn't get this error

**Comment by gamersindo1223** [comment]
nvm need to install some packages
apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev
